src/System.Management.Automation/engine/CommandSearcher.cs

```diff
@@ -1473,7 +1473,7 @@ private static CanDoPathLookupResult CanDoPathLookup(string possiblePath)
         /// <summary>
         /// The command name to search for.
         /// </summary>
-        private string _commandName;
+        private readonly string _commandName;
```

Comment:
For reviewers: this fixes a violation `IDE0044`: Add readonly modifier, enabled in #13880.